### ISSUE TITLE
Double_space: false positive?

### ISSUE THREAD

**Issue by DGRHicron** [abaplint/abaplint#3320]
![obraz](https://github.com/abaplint/abaplint/assets/143402755/39b4e17f-7405-48fc-afdc-b700f526ae8a)

**Comment by larshp** [comment]
hmm, might add an option to exclude certain keywords, https://rules.abaplint.org/double_space/

personally I line break after IMPORTING/EXPORTING/CHANGING/RETURNING keywords so I dont run into this problem

**Comment by DGRHicron** [comment]
I can't change the formatting as whole project is in that way and developers already used to it. I will wait for the keyword but I think it's a workaround rather than actual fix :)

**Comment by DGRHicron** [comment]
@larshp, will you be starting work on this in the near future?

**Comment by larshp** [comment]
try again, 2.112.1 published

**Comment by DGRHicron** [comment]
Unfortunately, still the same.
Here. Take this example. It can be replicated in the Playground.
```
REPORT zfoobar.
cl_gui_frontend_services=>get_platform( RECEIVING  platform             = platform
                                        EXCEPTIONS error_no_gui         = 1
                                                   cntl_error           = 2
                                                   not_supported_by_gui = 3
                                                   OTHERS               = 4 ).
```

**Comment by DGRHicron** [comment]
Works like a charm! Thanks!
